drivers/at86rf215/Makefile.dep

```diff
@@ -19,6 +19,10 @@ FEATURES_REQUIRED += periph_gpio
 FEATURES_REQUIRED += periph_gpio_irq
 FEATURES_REQUIRED += periph_spi
 
+ifeq (,$(filter gnrc_netif_pktq,$(USEMODULE)))
+  USEMODULE += at86rf215_blocking_send
+endif
```

Comment:
```suggestion
ifneq (,$(filter gnrc,$(USEMODULE)
  ifeq (,$(filter gnrc_netif_pktq,$(USEMODULE)))
    USEMODULE += at86rf215_blocking_send
  endif
endif
```

Otherwise you pull in blocking send when another stack that is not GNRC does support a queueing mechanism.

Reply:
Can be squashed directly IMHO. When applied, I agree with @maribu's ACK.